manifests/params.pp

```diff
@@ -3,36 +3,12 @@
 class foreman_scap_client::params {
   $ssl_dir = '/etc/puppetlabs/puppet/ssl'
```

Comment:
I'll note that it's a shame the SSL path settings are in extlib (https://github.com/voxpupuli/puppet-extlib/blob/master/lib/facter/extlib__puppet_config.rb) and not in stdlib (https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/lib/facter/puppet_settings.rb). That's something we regularly use in our modules and perhaps should make the effort to move into stdlib.